### PR TITLE
D8CORE-2282 Fix the admin toolbar menu link text

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Fixed the "All Media" toolbar link
- Changed the "master" reference to "main"

# Need Review By (Date)
- 7/8

# Urgency
- medium

# Steps to Test
1. Checkout this branch
1. clear caches
1. see the menu link the admin toolbar "All Media"

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: .github/workflows/tag_commits_to_master.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Tag
         uses: K-Phoen/semver-release-action@master
         with:
-          release_branch: master
+          release_branch: main
```
Comment (sherakama) [inline .github/workflows/tag_commits_to_master.yml:15]: ❤️  This change.

**Comment by sherakama** [comment]
I need to validate that site_managers can see it.

File: .github/workflows/tag_commits_to_master.yml (hunk 1 above)
Comment (jbickar) [reply .github/workflows/tag_commits_to_master.yml:15]: w00t.

**Comment by pookmish** [comment]
Site managers will get visibility to the item with the stanford profile pr. https://github.com/SU-SWS/stanford_profile/issues/246

**Pull request merged**